src/Backend/Modules/Blog/Domain/Comment.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="blog_comments")
+ * @ORM\Entity(repositoryClass="Backend\Modules\Blog\Domain\CommentRepository")
+ * @ORM\HasLifecycleCallbacks
+ */
+class Comment
+{
+    /**
+     * @var int
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var int
+     * @ORM\Column(type="integer", name="post_id")
```

Comment:
The name can use the default Doctrine naming scheme, we don't have to look at backwards compatibility anyway

Reply:
fixed in f7054a8635f84c330fe74e268a5490e6ba2edb2c